format/FlightSql.proto

```diff
@@ -867,6 +867,188 @@ enum SqlSupportsConvert {
   SQL_CONVERT_VARCHAR = 19;
 }
 
+/**
+ * It is an attribute that identifies the type of any object. It's
+ * used by JDBC and ODBC to expose the values defined by the DBMC.
```

Comment:
DBMC?

Reply:
Sorry, it was a typo. The correct is DBMS.

Fixed